### PR TITLE
Customize images philu

### PR DESCRIPTION
**Pull request by ahmed-faraz46**
- customized images for philu
- Merge Makefile.philu in main Makefile
- Backup main Makefile in Makefile.edx.ironwood
- Add make commands for up, start of studio and lms
move Makefile.philu to main Makefile, backup edx ironwood Makefile

### CODE REVIEW

**Review by emraan004 — APPROVED** [review]
👍

File: Makefile [hunk 1]
```diff
@@ -50,13 +50,13 @@ upgrade: ## Upgrade requirements with pip-tools
 		requirements/base.txt \
 
 dev.checkout: ## Check out "openedx-release/$OPENEDX_RELEASE" in each repo if set, "master" otherwise
-	./repo.sh checkout
+	./repo.sh checkout_philu
```
Comment (muhammadzeshanarif) [inline Makefile:56]: I think we should avoid duplicating the code for PhilU in our devstack.

File: Makefile [hunk 2]
```diff
@@ -73,6 +73,18 @@ dev.status: ## Prints the status of all git repositories
...
+	bash -c 'docker-compose -f docker-compose.yml -f docker-compose-host.yml start lms'
+
+dev.start.studio:
+	bash -c 'docker-compose -f docker-compose.yml -f docker-compose-host.yml start studio'
+
+dev.up.lms:
+	bash -c 'docker-compose -f docker-compose.yml -f docker-compose-host.yml up lms'
+
+dev.up.studio:
+	bash -c 'docker-compose -f docker-compose.yml -f docker-compose-host.yml up studio'
```
Comment (muhammadzeshanarif) [inline Makefile:89]: can we avoid this code repetition?

**Review by muhammadzeshanarif — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -50,13 +50,13 @@ upgrade: ## Upgrade requirements with pip-tools
 		requirements/base.txt \
 
 dev.checkout: ## Check out "openedx-release/$OPENEDX_RELEASE" in each repo if set, "master" otherwise
```
Comment (muhammadzeshanarif) [inline Makefile:55]: please update or remove comment, because this is not relevant to our devstack.

**Review by muhammadzeshanarif — CHANGES_REQUESTED** [review]

**Pull request closed without merging**